test/compilable/fix20964.d

```diff
@@ -0,0 +1,11 @@
+// https://issues.dlang.org/show_bug.cgi?id=20964
+
+uint foo(uint[3] m)
+{
+    auto p = m.ptr;
+    p += 3; // ok, p refers to the next right element after the block!
+    p -= 4; // ok, p refers to the next left element before the block!
```

Comment:
this is missing the `++p;` from the test case.

Reply:
Thanks, fixed